### DISCUSSION TITLE
permission denied to create extension "timescaledb"

### DISCUSSION THREAD

**Discussion by zenghao306** [filecoin-project/lily#315] [General] [answered]
2020-12-11T08:39:24.999Z	FATAL	visor	sentinel-visor/main.go:137	creating timescaledb extension: ERROR #42501 permission denied to create extension "timescaledb"

please tell me how to solve this problem,thanks

**Comment by iand** [comment, marked as answer]
Hi, You need to run visor with a user who has permissions to create extensions. By default the superuser has permissions. If you are using a packaged version of postgresql (such as the bitnami docker image) then your user may not have the necessary permissions.
